internal/batches/service/service.go

```diff
@@ -57,30 +59,41 @@ const sourcegraphVersionQuery = `query SourcegraphVersion {
 
 // getSourcegraphVersion queries the Sourcegraph GraphQL API to get the
 // current version of the Sourcegraph instance.
-func (svc *Service) getSourcegraphVersion(ctx context.Context) (string, error) {
+
+func (svc *Service) getSourcegraphVersionAndMaxChangesetsCount(ctx context.Context) (string, int, error) {
 	var result struct {
-		Site struct {
+		maxUnlicensedChangesets int
```

Comment:
Convention is usually to capitalize the start of property names, too:

```suggestion
		MaxUnlicensedChangesets int
```